tests/integration/cassettes/TestLiveThread_test_init.json

```diff
@@ -0,0 +1,164 @@
+{
+  "http_interactions": [
+    {
+      "recorded_at": "2016-12-03T04:01:48",
+      "request": {
+        "body": {
+          "encoding": "utf-8",
+          "string": "grant_type=client_credentials"
+        },
+        "headers": {
+          "Accept": "*/*",
+          "Accept-Encoding": "identity",
+          "Authorization": "Basic <BASIC_AUTH>",
+          "Connection": "keep-alive",
+          "Content-Length": "29",
+          "Content-Type": "application/x-www-form-urlencoded",
+          "User-Agent": "<USER_AGENT> PRAW/4.0.0 prawcore/0.3.0"
+        },
+        "method": "POST",
+        "uri": "https://www.reddit.com/api/v1/access_token"
+      },
+      "response": {
+        "body": {
+          "encoding": "UTF-8",
+          "string": "{\"access_token\": \"3ECWseFgsClB1rG7H0GMLSnHI9A\", \"token_type\": \"bearer\", \"expires_in\": 3600, \"scope\": \"*\"}"
```

Comment:
Yeah I believe most contributors will use dedicated account for testing, or revoke the token after testing, but I think it would be better if some safeguard exists.

> However, if you want to make a separate PR that filters out these access tokens transparently, I think that would be awesome.

Okay, I'll try to do that.